ansible/roles/products/hand-e/server/desktop-icons/tasks/main.yml

```diff
@@ -133,55 +114,10 @@
     command_to_run: "rqt"
     launch_terminal: "false"
 
-- name: Install the Documentation desktop icon for Dexterous Hand
-  include_role:
-    name: products/common/web-gui-icon
-  vars:
-    desktop_icon_png: "documentation_icon.png"
-    launch_script: "shadow_launcher_doc_exec.sh"
-    local_website_port_var: '7070'
-    desktop_icon_name: "{{ pub_icon_name }}"
-    desktop_icon_path: "{{ pub_icon_name }}"
-    launch_terminal: "false"
-    start_container_var: "true"
-    start_server_command_var: "roscd sr_dexterous_hand_documentation/html; python3 -m http.server {{ local_website_port_var }}"
-    preconditions_var: ""
-    live_website_url_var: "{{ pub_readthedocs }}"
-  when: 
-    - "'arm_' not in product"
-    - "'glove_' not in product"
-
-- name: Install the Documentation desktop icon for Dexterous Hand and Arm
-  include_role:
-    name: products/common/web-gui-icon
-  vars:
-    desktop_icon_png: "documentation_icon.png"
-    launch_script: "shadow_launcher_doc_exec.sh"
-    local_website_port_var: '7070'
-    desktop_icon_name: "Dexterous Hand and Arm Documentation"
-    desktop_icon_path: "Dexterous Hand and Arm Documentation"
-    launch_terminal: "false"
-    start_container_var: "true"
-    start_server_command_var: "roscd sr_dexterous_hand_and_arm_documentation/html; python3 -m http.server {{ local_website_port_var }}"
-    preconditions_var: ""
-    live_website_url_var: "{{ arm_hand_readthedocs_link }}"
-  when: "'arm_' in product"
-
-- name: Install the Documentation desktop icon for Dexterous Hand and Glove
+- name: Include documentation role
   include_role:
-    name: products/common/web-gui-icon
-  vars:
-    desktop_icon_png: "documentation_icon.png"
-    launch_script: "shadow_launcher_doc_exec.sh"
-    local_website_port_var: '7070'
-    desktop_icon_name: "Dexterous Hand and Glove Documentation"
-    desktop_icon_path: "Dexterous Hand and Glove Documentation"
-    launch_terminal: "false"
-    start_container_var: "true"
-    start_server_command_var: "roscd sr_dexterous_hand_glove_documentation/html; python3 -m http.server {{ local_website_port_var }}"
-    preconditions_var: ""
-    live_website_url_var: "{{ glove_hand_readthedocs_link }}"
-  when: "'glove_' in product"
+    name: products/common/documentation
+  when: customer_key is defined and customer_key | length > 0
```

Comment:
so there won't be a documentation file if the customer key is not specified? so if someone goes to our documentation,and uses the oneliner to install the simulation configured imaged, they won't have the documentation?

Reply:
had a brief discussion with @BenStarmerSmith, this is not a preferred way as this will cause issues with people who are not customers but are going to install our software via the oneliner. The best approach, for now, is to have a default customer_key or other authentication to 'pull-only' the dex hand documentation but needs more investigation on a better way.

Reply:
Followup story: https://shadowrobot.atlassian.net/browse/SP-436